packages/front-end/components/Forms/CodeTextArea.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactNode } from "react";
+import Field, { FieldProps } from "./Field";
+import AceEditor from "react-ace";
+import "ace-builds/src-noconflict/mode-sql";
+import "ace-builds/src-noconflict/mode-javascript";
+import "ace-builds/src-noconflict/mode-python";
+import "ace-builds/src-noconflict/mode-yaml";
+import "ace-builds/src-noconflict/mode-json";
+import "ace-builds/src-noconflict/theme-textmate";
+
+export type Props = Omit<
+  FieldProps,
+  "value" | "onChange" | "options" | "multi" | "initialOption"
+> & {
+  syntax: string;
+  placeholder?: string;
+  currentValue: string;
+  setValue: (value: string) => void;
+  codeTextAreaHeight?: string;
```

Comment:
This doesn't seem like it's being used anywhere. I would remove it or at least change the name to just height.

Reply:
10-4. I've just removed it for now. I added it as I have a feeling it might be beneficial when we started updating the other language text fields. But if we need it, we can easily add it back when we do that work.